src/components/skills-quiz/data/service.js

```diff
@@ -2,20 +2,11 @@ import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
 import { getConfig } from '@edx/frontend-platform/config';
 import { logError } from '@edx/frontend-platform/logging';
 
-export function fetchSkillsId(skills) {
-  const queryParams = new URLSearchParams({
-    name: skills,
-  });
-  const config = getConfig();
-  const url = `${config.DISCOVERY_API_BASE_URL}/taxonomy/api/v1/skills/?${queryParams.toString()}`;
-  return getAuthenticatedHttpClient().get(url);
-}
-
-export function postSkillsGoalsAndJobsUserSelected(goal, skillsId, interestedJobsId, currentJobRoleId) {
+export function postSkillsGoalsAndJobsUserSelected(goal, interestedJobsId, currentJobRoleId) {
   const options = {
     goal,
     current_job: currentJobRoleId ? currentJobRoleId[0] : null,
-    skills: skillsId,
+    skills: [],
```

Comment:
Can't we just simply remove `skills` from options?

Reply:
For now, we can't do so, in taxonomy-connector, API is expecting this field. After removing it from the backend too, we can remove this from here easily.